### DISCUSSION TITLE
Anzeigen des Datums der letzten Impfung bei der “Kontrolle”

### DISCUSSION THREAD

**Discussion by schmdjhnnsn** [Digitaler-Impfnachweis/certification-apis#220] [Ideas]
Die App geigt nicht das Datum der zuletzt erfolgten Impfung an, nachdem ein Zertifikat geprüft wurde.
Das führt dazu, dass die Kontrolle meist nicht kontaktlos stattfindet, sondern dass der Konteollierende auf dem Handy des geimpften herumtippt.

Warum nicht neben der Gültigkeit auch das Datum oder die Anzahl der Tage seit der letzten Inpfung direkt ausgeben?!

**Comment by Ein-Tim** [comment]
This question is indirect answered in https://github.com/Digitaler-Impfnachweis/covpass-ios/issues/60 where @jleufgen stated that they are not allowed to show whether one was recovered or vaccinated due to data privacy. 
Showing the date of vaccination would reveal that one was vaccinated.
